### PR TITLE
Update Apache Daffodil KEYS

### PR DESCRIPTION
**Pull request by pkatlic**

### CODE REVIEW

File: KEYS
Comment (stevedlawrence) [inline KEYS:362]: Downloading the keys file and running `gpg --list-packets KEYS | grep -E '(user ID|digest algo)'`, it looks like your key is algorithm 8, which is sha256. Although that should work fine, ASF prefers sha-512. It might be best to regenerate your key with sha512 in case apache or anything ever deprecates use of sha256. We had an issue where RPM deprecated sha1 and we had to regenerate some keys, so it's not unheard of.

I think the settings mentioned in "Add some lines to your gpg.conf file:" in this file will cause it to generate sha512.

**Review by stevedlawrence — CHANGES_REQUESTED** [review]

**Review by stevedlawrence — APPROVED** [review]
+1, looks good!

**Comment by stevedlawrence** [comment]
We usually require 2 +1's from a committer/PMC member before merging a pull request. This one is small enough that it's not a big deal that you merged it, but for future PR's please wait for 2 +1's.

**Review by mbeckerle — COMMENTED** [review]
+1

**Comment by pkatlic** [comment]
@stevedlawrence Got it, will make sure to wait for the 2 +1s on future PRs.

**Pull request merged**
